packages/plugin-rsc/src/plugin.ts

```diff
@@ -1745,7 +1831,7 @@ function generateResourcesCode(depsCode: string) {
   const ResourcesFn = (React: typeof import('react'), deps: AssetDeps) => {
     return function Resources() {
       return React.createElement(React.Fragment, null, [
-        ...deps.css.map((href: string) =>
+        ...deps.css.map((href) =>
```

Comment:
I think the typing is a bit confused here. Isn't `href` is known to be `string` here already? (i.e. runtime coming from `serverResources` is resolved already)